meta-intel-extras/recipes-support/grub/grub-efi/grubenv-copy.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
is service enabled?

Reply:
Yes. The following line in grub-efi recipe makes sure that it is enabled

SYSTEMD_SERVICE_${PN} = "grubenv-copy.service"